frontend/src/Theme.css

```diff
@@ -107,7 +107,7 @@
 	--chat-alerted-text-colour: teal;
 	--chat-triggered-text-colour: var(--error-colour);
 	--chat-button-background-colour: #eee;
-	--chat-button-background-colour-hover: #dedede;
+	--chat-button-background-colour-hover: #cecece;
```

Comment:
I assume you've checked contrast is still ok, on hover?

Reply:
yup, all good